core/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -37,24 +37,25 @@ class BasicWorkflow extends EnsimeSpec
 
           project ! TypecheckModule(EnsimeProjectId("testing_simple", "compile"))
           expectMsg(VoidResponse)
-          all(asyncHelper.receiveN(2)) should matchPattern {
+          all(asyncHelper.receiveN(3)) should matchPattern {
```

Comment:
heh, nice catch. I actually tried something like this lower down, but it was after the fishing so it was missed. We should probably rationalise our use of `fish` everywhere.